ablator/main/model/wrapper.py

```diff
@@ -615,6 +962,14 @@ def checkpoint(self, is_best=False):
         )
 
     def save_dict(self) -> dict[str, ty.Any]:
+        """
+        Save the current state of the trainer.
```

Comment:
"state" includes model parameters, and current states of the optimizer, the scaler, and the scheduler